geonode/maps/migrations/0033_map_blob_migration.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
We cannot assume the `mapstore2 adapter` tables are present on GeoNode core, since the client module is pluggable.

This migration must be moved to the `geonode-mapstore-client` module instead.